{"name": "API 키가 잘못되었을 때 401 Unauthorized 확인", "status": "passed", "start": 1743373044389, "stop": 1743373044423, "uuid": "0569f3e1-8cca-4c20-8356-328b78266c90", "historyId": "a386eac7b0b41582b56dc8e54e039987", "testCaseId": "a386eac7b0b41582b56dc8e54e039987", "fullName": "src.tests.api_tests.test_moive_invalid_api_key#test_movie_invalid_api_key", "labels": [{"name": "feature", "value": "예외 케이스"}, {"name": "story", "value": "잘못된 API 키"}, {"name": "parentSuite", "value": "src.tests.api_tests"}, {"name": "suite", "value": "test_moive_invalid_api_key"}, {"name": "host", "value": "fv-az1043-103"}, {"name": "thread", "value": "3619-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.api_tests.test_moive_invalid_api_key"}]}